recipes/tachyon/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd unix
+case "$(uname)" in
+    Darwin) target=macosx-x86_64-thr;;
+    Linux) target=linux-64-thr;;
+esac
+
+make $target -j${CPU_COUNT}
+
+cd ..
+cp compile/*/tachyon "$PREFIX/bin"
```

Comment:
there are no `make check` or `make install` targets. I've no idea how to contact upstream though. Home url is broken at the moment and I've used the tarball from fedora project.